src/main/java/net/liquidpineapple/pang/gui/Board.java

```diff
@@ -1,12 +1,38 @@
 package net.liquidpineapple.pang.gui;
 
 import javax.swing.*;
+import java.awt.*;
 
 /**
  * @author Jurriaan Den Toonder<jurriaan.toonder@liquidpineapple.net>
  * @date 2016/09/06.
  */
 public class Board extends JPanel {
 
-    public Board() {}
+    private JPanel panel = new JPanel();
```

Comment:
Class Board extends JPanel. Why do we need to create a field panel here. Can't we use `this` ?